src/utils/rdtsc.cpp

```diff
@@ -69,7 +69,8 @@ bool get_cpu_hz(double &hz_min, double &hz_max)
 		if (rc != 1) {
 			continue;
 		}
-		if (mhz_tmp == 0.0) {
+		// Coverity warning : comparing floating point with == or != is unsafe
+		if (0 == static_cast<int>(mhz_tmp * 100000)) {
```

Comment:
converting to `int` is not the best way to do this.

1. Use a dedicate `bool do_once` flag to modify the first off min + max values

2. Use something like: `bool compareDouble(doubel a, doube b) {return fabs(a - b) < std::numeric_limits<double>::epsilon(); } `
(See referance in: http://stackoverflow.com/questions/17333/most-effective-way-for-float-and-double-comparison)